app/src/main/java/nl/hogeschoolrotterdam/projectb/data/room/AppDatabase.java

```diff
@@ -12,7 +12,7 @@
  * Created by maartendegoede on 23/04/2019.
  * Copyright © 2019 Anass El Mahdaoui, Hicham El Marzgioui, Wesley de Man, Maarten de Goede all rights reserved.
  */
-@Database(entities = {Memory.class, Media.class, Image.class, Video.class}, version = 1)
+@Database(entities = {Memory.class, Media.class, Image.class, Video.class}, version = 2)
```

Comment:
I'll figure out if this version update is really necessary and how to handle it. Don't merge anything just yet.